cmd/login/login_test.go

```diff
@@ -165,3 +209,13 @@ func TestLogin(t *testing.T) {
 		require.True(t, gock.IsDone())
 	})
 }
+
+func newTestInsecureServer(t testing.TB, path string, h func(w http.ResponseWriter, r *http.Request)) *httptest.Server {
+	t.Helper()
+
+	mux := http.NewServeMux()
+	server := httptest.NewServer(mux)
```

Comment:
Server should not have a self signed cert?

Reply:
Yep